tests/dem/post_collision_velocity.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::Coefficient of restitution is 0.7 and the velocity of particle before collision is 0.1, the velocity of particle after collision is: 0.0662483
```

Comment:
Shouldnt it be 0.07 and not 0.066? Why is there such a difference? Does it go down when you decrease the time step?